docs/HowTo/Configure/Genesis-File.md

```diff
@@ -11,7 +11,8 @@ For Ethereum MainNet and public testnets (for example, Rinkeby) the genesis conf
 definition is in Besu and used when specifying a public network using the
 [`--network`](../../Reference/CLI/CLI-Syntax.md#network) command line option.
 
-For private networks, create a JSON genesis file and specify the genesis file using the
+For private networks, which you might want to [configure with free gas](FreeGas.md) for example,
+create a JSON genesis file and then specify the genesis file using the
```

Comment:
This is quite a long sentence now - 31 words and include more than once concept.  I'd suggest splitting into two.  

I think the point about free gas and linking to the free gas topic might belong more naturally with the next paragraph about network wide settings being set in the genesis file.

Reply:
Yes, very good point. Have reverted this sentence, then included the link in the next sentence.